### PR TITLE
Restore Franka core tasks on the shared Menagerie asset

### PR DESCRIPTION
**Pull request by maxkra15**
# Description

Restore the core Franka task contracts on one shared, flat Menagerie asset after the asset and
teleoperation changes regressed policy compatibility and PhysX startup time.

This PR:

- makes the flat asset canonical, with explicit `Physics` and `Colliders` variants and
  `gripper_only` as the task default;
- aligns actuator, mimic-finger, controller, frame-resolution, reset-clearance, and solver contracts
  across PhysX and Newton;
- keeps full-arm contacts an explicit task/domain choice;
- adds the minimum Reorient bootstrap needed for scratch training: shape-aware pregrasp resets,
  ADR-scaled pose goals, and a single active-finger gripper action; and
- includes the latest `develop` changes. Camera policies remain out of scope.

No USD or checkpoint binaries are committed here. Review artifacts are available at:

```text
omniverse://isaac-dev.ov.nvidia.com/Isaac/IsaacLab/Robots/FrankaEmika/franka_panda.usda
omniverse://isaac-dev.ov.nvidia.com/Isaac/IsaacLab/PretrainedCheckpoints/rsl_rl/
```

Public S3 publication remains a post-review step.

## Current results

The clean asset halves 8,192-environment Reach/PhysX creation time (72.55 s to 36.74 s) without a
success regression. Fresh readback evaluation of the supported non-camera checkpoints remained above
the agreed threshold: Reach 99.26% PhysX / 96.88% Newton, Drawer 100.00% / 98.44%, Soft Lift 97.67% /
93.48%, Cloth Lift 99.84% / 94.15%, Cable Lift 98.83% Newton, and rigid Lift 87.97% PhysX.

Fresh scratch Lift training on current `develop` converged independently on both backends:

- [Newton: 98.99% success, 97.91% ADR](https://wandb.ai/nvidia-isaac/franka_backend_parity/runs/develop-a135342-lift-newton-scratch-8gpu-s42)
- [Isaac Sim PhysX: 97.00% success, 96.07% ADR](https://wandb.ai/nvidia-isaac/franka_backend_parity/runs/develop-a135342-lift-isaacsim-physx-scratch-1gpu-s42)

The final Reorient config reached 94% initial exact success locally and 78.2% cumulative success at
iteration 113 while ADR was still rising (32.7%). A full scratch qualification is now running on eight
GPUs:

- [OSMO workflow](https://us-west-2-aws.osmo.nvidia.com/workflows/franka-reorient-pr7829-18fc5f644-newton-8gpu-s42-1)
- [W&B run](https://wandb.ai/nvidia-isaac/franka_backend_parity/runs/franka-reorient-pr7829-18fc5f644-newton-scratch-8gpu-s42)

OvPhysX training remains blocked by the separate runtime regression introduced by #7861; the same
source, task, and asset learn with the previous OvPhysX runtime.

## Validation

```bash
uv run --frozen python -m pytest \
  source/isaaclab_tasks/test/core/test_lift_env_cfg.py \
  source/isaaclab_assets/test/test_franka_asset_contract.py -q
uv run --frozen isaaclab -f
```

Result: 14 focused tests passed and formatting passed. The current head is `18fc5f644` and GitHub CI is
in progress.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (existing functionality will not work without modification)
- [ ] Documentation update

## Release backport

- [ ]  Backport this pull request to the active release branch after it merges into `develop`

## Checklist

- [x] I have read and understood the contribution guidelines
- [x] I have run the pre-commit checks with `uv run isaaclab -f`
- [x] I have made corresponding documentation/changelog changes
- [x] My changes generate no new warnings
- [x] I have added tests that prove the fixes are effective
- [x] I have added a changelog fragment for every touched source package
- [x] My name already exists in `CONTRIBUTORS.md`

### CODE REVIEW

File: source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py [hunk 1]
```diff
@@ -175,6 +180,7 @@ def IO_descriptor(self) -> GenericActionIODescriptor:
         self._IO_descriptor.body_name = self._body_name
         self._IO_descriptor.joint_names = self._joint_names
         self._IO_descriptor.scale = self._scale
+        self._IO_descriptor.offset = self._offset[0].detach().cpu().numpy().tolist()
```
Comment (yufengsjtu) [inline source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py]: align with scale? (last line)

File: source/isaaclab/isaaclab/envs/mdp/actions/actions_cfg.py [hunk 2]
```diff
@@ -325,6 +325,11 @@ class OffsetCfg:
     """Offset of target frame w.r.t. to the body frame. Defaults to None, in which case no offset is applied."""
     scale: float | tuple[float, ...] = 1.0
     """Scale factor for the action. Defaults to 1.0."""
+    offset: float | tuple[float, ...] = 0.0
+    """Offset applied to the scaled action. Defaults to 0.0.
```
Comment (StafaH) [inline source/isaaclab/isaaclab/envs/mdp/actions/actions_cfg.py]: Let's make comment one line like the rest

File: source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py (hunk 1 above)
Comment (StafaH) [inline source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py]: this .detach().cpu().numpy().tolist() looks very suspicious, what's going on, is there a better way here?

File: source/isaaclab/test/envs/test_diffik_action_processing.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+import pytest
+import torch
+
+from isaaclab.envs.mdp.actions.task_space_actions import DifferentialInverseKinematicsAction
+
+pytestmark = pytest.mark.unit
+
+
+class _ControllerStub:
```
Comment (StafaH) [inline source/isaaclab/test/envs/test_diffik_action_processing.py]: If using the original is fast, let's use that instead of mocks/stubs. Let's modify the tests to be lean and fast, but still use original classes/datastructures where possible.

File: source/isaaclab_assets/test/test_franka_cfg.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2026, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
```
Comment (StafaH) [inline source/isaaclab_assets/test/test_franka_cfg.py]: This test file is not testing anything worthwhile, just a bunch of asserts for hardcoded values. let's remove (and also avoid this here and elsewhere)

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/rewards.py [hunk 5]
```diff
@@ -164,7 +165,7 @@ def __call__(
             reward = ((1 - torch.tanh(pos_dist / pos_std)) ** 2) * contact_mask.float()
             self.succeeded |= (pos_dist < pos_std) & contact_mask
 
-        return reward
+        return torch.where(torch.isfinite(reward), reward, torch.zeros_like(reward))
```
Comment (StafaH) [inline source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/rewards.py]: Such a guard is expensive, isfinite means you are expecting values to blow up, usually inf is bad and is not the root of the problem, maybe you can try to figure out why the reward would blow up to inf and solve it there?

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py [hunk 6]
```diff
@@ -61,7 +61,12 @@ def __call__(
                 self._cached_axis[i] = bounds
 
         pos_w = self._object.data.root_pos_w.torch
-        return ((pos_w < self._lower) | (pos_w > self._upper)).any(dim=1)
+        quat_w = self._object.data.root_quat_w.torch
```
Comment (StafaH) [inline source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py]: Since these arrays are ProxyArrays, they should be able to pass as argument directly to torch functions (usually not in all cases, but worth a try)

File: source/isaaclab_tasks/test/core/test_franka_shared_asset_presets.py [hunk 7]
```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2026, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
```
Comment (StafaH) [inline source/isaaclab_tasks/test/core/test_franka_shared_asset_presets.py]: Not sure how meaningful these tests are, similar comment to before. Can we cleanup all the tests in this PR and make sure its lean and only keep the good tests

File: source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py (hunk 1 above)
Comment (maxkra15) [reply source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py]: yes, it now matches scale.

File: source/isaaclab/isaaclab/envs/mdp/actions/actions_cfg.py (hunk 2 above)
Comment (maxkra15) [reply source/isaaclab/isaaclab/envs/mdp/actions/actions_cfg.py]: done.

File: source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py (hunk 1 above)
Comment (maxkra15) [reply source/isaaclab/isaaclab/envs/mdp/actions/task_space_actions.py]: yes, it now passes the tensor directly.

File: source/isaaclab/test/envs/test_diffik_action_processing.py (hunk 3 above)
Comment (maxkra15) [reply source/isaaclab/test/envs/test_diffik_action_processing.py]: done, it now uses the real config and controller.

File: source/isaaclab_assets/test/test_franka_cfg.py (hunk 4 above)
Comment (maxkra15) [reply source/isaaclab_assets/test/test_franka_cfg.py]: removed.

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/rewards.py (hunk 5 above)
Comment (maxkra15) [reply source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/rewards.py]: removed; the reward is bounded, so bad state now stays visible.

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py (hunk 6 above)
Comment (maxkra15) [reply source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py]: yes, they now pass directly.

File: source/isaaclab_tasks/test/core/test_franka_shared_asset_presets.py (hunk 7 above)
Comment (maxkra15) [reply source/isaaclab_tasks/test/core/test_franka_shared_asset_presets.py]: done, the redundant tests are removed.

**Comment by maxkra15** [comment]
run-ci

File: source/isaaclab_assets/isaaclab_assets/robots/franka.py [hunk 8]
```diff
@@ -73,38 +74,54 @@
...
 )
-"""Configuration of Franka Emika Panda robot."""
+"""Configuration of the legacy Franka Emika Panda robot asset."""
 
 
-FRANKA_PANDA_MENAGERIE_CFG = FRANKA_PANDA_CFG.copy()
-FRANKA_PANDA_MENAGERIE_CFG.spawn.usd_path = f"{ISAACLAB_NUCLEUS_DIR}/Robots/FrankaEmika/franka_panda.usda"
-FRANKA_PANDA_MENAGERIE_CFG.actuators = {
+FRANKA_PANDA_CFG = FRANKA_PANDA_LEGACY_CFG.copy()
+FRANKA_PANDA_CFG.spawn.usd_path = f"{ISAACLAB_NUCLEUS_DIR}/Robots/FrankaEmika/franka_panda.usda"
```
[the remark this answers is not in the document]
Comment (maxkra15) [reply source/isaaclab_assets/isaaclab_assets/robots/franka.py:81]: fixed; contrib and teleop paths now support both asset hierarchies.

File: source/isaaclab_assets/isaaclab_assets/robots/franka.py [hunk 9]
```diff
@@ -73,38 +74,54 @@
...
+FRANKA_PANDA_CFG.spawn.variants = {"Physics": "physx", "Colliders": "gripper_only"}
```
[the remark this answers is not in the document]
Comment (maxkra15) [reply source/isaaclab_assets/isaaclab_assets/robots/franka.py:82]: fixed; direct newton callers now select the mujoco payload explicitly.

File: source/isaaclab_assets/isaaclab_assets/robots/franka.py [hunk 10]
```diff
@@ -73,38 +74,54 @@
...
+FRANKA_PANDA_CFG.spawn.articulation_props.enabled_self_collisions = False
+FRANKA_PANDA_CFG.actuators = {
```
[the remark this answers is not in the document]
Comment (maxkra15) [reply source/isaaclab_assets/isaaclab_assets/robots/franka.py:84]: fixed; the major changelog now documents the actuator-key migration and legacy escape hatch.

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py [hunk 11]
```diff
@@ -60,8 +60,13 @@ def __call__(
                 self._upper[:, i] = self._origins[:, i] + hi
                 self._cached_axis[i] = bounds
 
-        pos_w = self._object.data.root_pos_w.torch
-        return ((pos_w < self._lower) | (pos_w > self._upper)).any(dim=1)
+        pos_w = self._object.data.root_pos_w
+        quat_w = self._object.data.root_quat_w
+        vel_w = self._object.data.root_vel_w
```
[the remark this answers is not in the document]
Comment (maxkra15) [reply source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py]: fixed; these reads now use explicit .torch views.

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py [hunk 12]
```diff
@@ -60,8 +60,13 @@ def __call__(
                 self._upper[:, i] = self._origins[:, i] + hi
                 self._cached_axis[i] = bounds
 
-        pos_w = self._object.data.root_pos_w.torch
-        return ((pos_w < self._lower) | (pos_w > self._upper)).any(dim=1)
+        pos_w = self._object.data.root_pos_w
```
[the remark this answers is not in the document]
Comment (maxkra15) [reply source/isaaclab_tasks/isaaclab_tasks/core/lift/mdp/terminations.py]: fixed; all three reads now use explicit .torch views.

File: source/isaaclab_assets/isaaclab_assets/robots/franka.py [hunk 13]
```diff
@@ -73,38 +74,54 @@
...
+"""Configuration of the legacy Franka Emika Panda robot asset."""
 
 
-FRANKA_PANDA_MENAGERIE_CFG = FRANKA_PANDA_CFG.copy()
-FRANKA_PANDA_MENAGERIE_CFG.spawn.usd_path = f"{ISAACLAB_NUCLEUS_DIR}/Robots/FrankaEmika/franka_panda.usda"
-FRANKA_PANDA_MENAGERIE_CFG.actuators = {
+FRANKA_PANDA_CFG = FRANKA_PANDA_LEGACY_CFG.copy()
+FRANKA_PANDA_CFG.spawn.usd_path = f"{ISAACLAB_NUCLEUS_DIR}/Robots/FrankaEmika/franka_panda.usda"
+FRANKA_PANDA_CFG.spawn.variants = {"Physics": "physx", "Colliders": "gripper_only"}
+FRANKA_PANDA_CFG.spawn.articulation_props.enabled_self_collisions = False
```
[the remark this answers is not in the document]
Comment (maxkra15) [reply source/isaaclab_assets/isaaclab_assets/robots/franka.py:83]: fixed; robotiq now derives from the legacy config and keeps self-collisions enabled.

**Comment by maxkra15** [comment]
run-ci
